.github/workflows/unit-test.yml

```
@@ -45,7 +45,57 @@ jobs:
         npx lerna run build
 
     - name: Running the tests
-      run: npm run test-unit -- --ci --maxWorkers=2 --cacheDirectory="$HOME/.jest-cache"
+      run: |
+        npm run test-unit -- --ci --maxWorkers=2 --cacheDirectory="$HOME/.jest-cache"
```

Nitpicking (revert to one line)
```suggestion
      run: npm run test-unit -- --ci --maxWorkers=2 --cacheDirectory="$HOME/.jest-cache"
```